modules/gapi/test/gapi_array_tests.cpp

```diff
@@ -164,6 +177,25 @@ TEST(GArray, TestIntermediateOutput)
     EXPECT_EQ(10,  out_count[0]);
 }
 
+TEST(GArray, GArrayInitialization)
```

Comment:
Mb name the test case more specifically? Like, `GArrayConstValueInitialization`
Who knows, perhaps there'll be other initializations in the future...

Reply:
M B